apps/server/src/services/storageClient/redis.ts

```diff
@@ -8,6 +8,12 @@ import { PREFERENCES_REDIS_KEY } from "./constants";
 
 const redisClient = createClient({
   url: config.REDIS_SERVER,
+  ...(config.REDIS_ENABLE_TLS && {
+    socket: {
+      tls: true,
+      rejectUnauthorized: false,
+    },
+  }),
```

Comment:
This config makes Redis work in heroku, but it breaks things locally and in the github workflows, that's why I added an optional env variable which I could just set in Heroku to get it to work.